includes/multi-currency/AdminNotices.php

```diff
@@ -21,9 +21,11 @@ class AdminNotices {
 	private $notices = [];
 
 	/**
-	 * Constructor
+	 * Initializes this class's WP hooks.
+	 *
+	 * @return void
 	 */
-	public function __construct() {
+	public function init_hooks() {
```

Comment:
This class only had hooks in the `__construct` method, so it was replaced with `init_hooks`.